pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launcher.cdn.ankama.com/installers/production/Ankama%20Launcher-Setup-x86_64.AppImage";
```

Change the url for the following one:
```suggestion
    url = "https://launcher.cdn.ankama.com/installers/production/Dofus-Setup-x86_64.AppImage";
```
I found this one inspecting the Dofus page on the network tab. I clicked the download button while inspecting the network and got this request URL eheheheh. And it works perfectly :)